app/src/app/list/polls-table/polls-table.component.sass

```diff
@@ -62,24 +66,25 @@ table
       margin-bottom: $medium-space
       padding: $small-space
       td
-        border: solid 1px #ddd
         padding: $tiny-space
         display: block
         text-align: right
         text-overflow: ellipsis
         overflow: hidden
         white-space: nowrap
+      td.title
+        border-bottom: solid 2px #ddd
+        text-align: left
+        font-weight: bolder
+      td.round,  td.deadline,  td.terminated
+        font-size: smaller
+        text-align: right
         &:before
           content: attr(data-label)
           float: left
           font-weight: bold
           padding-right: 1.5em
-      td.round,  td.deadline,  td.terminated
-        text-align: right
-      td.action
+      td.action-manage
+        display: block
```

Comment:
Seems wrong to me. This results in wrong aligment.
![image](https://user-images.githubusercontent.com/1835665/121160709-9eaade80-c84c-11eb-9a9d-f4f077d5a0a4.png)
I suggest:
```suggestion
        justify-content: end
```